### ISSUE TITLE
wordy: update tests to v1.5.0

### ISSUE THREAD

**Issue by cmccandless** [exercism/python#1624]
The [test suite](https://github.com/exercism/python/blob/master/exercises/wordy/wordy_test.py) for wordy is out of date and needs updated to conform to the [latest canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/wordy/canonical-data.json).

**Comment by CometS1** [comment]
I am trying to implement the updates to the test suite. However, when attempting to run the check for the exercises I get the following error:
![image](https://user-images.githubusercontent.com/25065296/49654129-ff030000-fa04-11e8-8f1e-d6a2b74a5f26.png)
Do you know the solution?

**Comment by cmccandless** [comment]
Alternatively, you can temporarily change the import in the test file to use `example.py` instead:
```
# Replace
from wordy import calculate
# with
from example import calculate
```

Then just run `pytest` normally in the `exercises/wordy` directory.

**Comment by cmccandless** [comment]
@CometS1 #1626 has been merged to address the issue with `check-exercises.py`. Please merge the latest changes into your branch.

**Comment by cmccandless** [comment]
One last note: I highly recommend testing your changes locally with `python test/check-exercises.py wordy`, as some exercises (like `bank-account`) may take some time to run. Travis-CI builds will test against every exercise so you don't necessarily have to do so.

**Comment by CometS1** [comment]
What is needed to update wordy to its current version?

**Comment by cmccandless** [comment]
@CometS1 
Looks like this was already resolved in #1608 and #1610, but this didn't get closed. Sorry for the confusion, please check out our other open issues.
